--- a/m4/efl.m4
+++ b/m4/efl.m4
@@ -113,7 +113,7 @@ dnl TODO: warning - lt_cur:
 dnl the previous code assumed v_maj + v_min, but this will be a problem when
 dnl we bump v_maj and reset v_min. 1 + 7 == 7 + 1, so if v_maj is bumped
 dnl we multiply it by 100.
-m4_define([lt_cur], m4_if(m4_cmp(v_maj, 1), 0, m4_eval(v_maj + v_min), m4_eval(v_maj * 100 + v_min)))dnl
+m4_define([lt_cur], m4_if(m4_cmp(v_maj, 1), 0, m4_eval(v_maj + v_min + 1), m4_eval(v_maj * 100 + v_min)))dnl
 m4_define([lt_rev], v_mic)dnl
 m4_define([lt_age], v_min)dnl
 dnl
